### ISSUE TITLE
Error with Multiple Unit Train Control

### ISSUE THREAD

**Issue by 0ptera** [Choumiko/FARL#103]
Converting existing farl to MU by decoupling and recoupling a locomotive with FARL equipment causes farl to throw the error:

```
1468.114 Script @__FARL__/lib_control.lua:21: stack traceback:
	__FARL__/lib_control.lua:21: in function 'debugDump'
	__FARL__/control.lua:116: in function <__FARL__/control.lua:80>
	[C]: in function 'pcall'
	__FARL__/control.lua:80: in function <__FARL__/control.lua:79>
1468.115 Script @__FARL__/lib_control.lua:22: 114782014 Unexpected error: LuaBurner API call when LuaBurner was invalid.
stack traceback:
	__FARL__/FARL.lua:1244: in function 'cruiseControl'
	__FARL__/FARL.lua:440: in function 'update'
	__FARL__/control.lua:108: in function <__FARL__/control.lua:107>
	[C]: in function 'pcall'
	__FARL__/control.lua:107: in function <__FARL__/control.lua:80>
	[C]: in function 'pcall'
	__FARL__/control.lua:80: in function <__FARL__/control.lua:79>
```

[factorio-current.log](https://github.com/Choumiko/FARL/files/3053577/factorio-current.log)

**Comment by Choumiko** [comment]
Couldn't reproduce but it should be fixed anyways, since integration with MU was improved

**Comment by 0ptera** [comment]
The error seems to be fixed in 3.1.5 for me as well.
